binscripts/rvm-installer

```diff
@@ -793,17 +793,22 @@ rvm_install_validate_volume_mount_mode()
   do
       if [[ -d $path ]]
       then
+        test_exec=$(mktemp $path/rvm-exec-test.XXXXXX)
+        echo '#!/bin/sh' > "$test_exec"
+        chmod +x "$test_exec"
         partition=`df -P $path | awk 'END{print $1}'`
 
-        if mount | grep "$partition .*noexec.*" >/dev/null 2>&1
+        if ! "$test_exec"
         then
+          rm -f "$test_exec"
           printf "%b" "
 It looks that partition holding RVM destination location ${rvm_path}
 is mounted in *noexec* mode, which prevents RVM from working correctly.
 Please re-mount partition ${partition} without the noexec option."
```

Comment:
we could rephrase it with `Please re-mount ${path} without the noexec option.` - and drop the partition detection - one less thing to think about.